src/main/loadscreen.js

```diff
@@ -56,7 +57,7 @@ function loadScript() {
         loadScript();
       } else {
         loading = false;
-        start();
+        main.start();
```

Comment:
Change this to `window.start`, the main bundle attaches the start function to the window which this should execute. It's unfortunately the only way we have right now to communicate across bundles.